fakes/v0.0/fakes-v0.0.graphql

```diff
@@ -0,0 +1,75 @@
+enum Type {
+    zipCode
+    city
+    streetName
+    streetAddress
+    secondaryAddress
+    country
+    countryCode
+    state
+    stateAbbr
+    latitude
+    longitude
+
+    colorName
+    productName
+    money
+    productMaterial
+
+    companyName
+    companyCatchPhrase
+    companyBS
+
+    date
+    pastDate
+    futureDate
+
+    currencyCode
+    currencyName
+    currencySymbol
+    internationalBankAccountNumber
+    bankIdentifierCode
+
+    hackerAbbreviation
+
+    imageUrl
+    avatarUrl
+    email
+    url
+    domainName
+    ipv4Address
+    ipv6Address
+    userAgent
+    colorHex
+    macAddress
+    password
+    lorem
+
+    firstName
+    lastName
+    fullName
+    jobTitle
+
+    phoneNumber
+
+    number
+    uuid
+    word
+    words
+    locale
+
+    filename
+    mimeType
+    fileExtension
+    semver
+}
+
+directive @fake(
+    type: Type!
+) on FIELD_DEFINITION | SCALAR
+
+directive @listSize(level: Int! = 0, min: Int!, max: Int!) repeatable on FIELD_DEFINITION
```

Comment:
Do we worry this will be conflated with the cost directives?

Reply:
https://ibm.github.io/graphql-specs/cost-spec.html#sec-The-List-Size-Directive

```graphql
directive @listSize(
  assumedSize: Int,
  slicingArguments: [String!],
  sizedFields: [String!],
  requireOneSlicingArgument: Boolean = true
) on FIELD_DEFINITION
```

Thanks for flagging this. That's a valid concern. Not sure what else to use, maybe `@fakeList(minSize:3, maxSize: 5)`?

Reply:
I think the `@fakeList` is probably okay here. You _might_ be able to overload the proposed directive with a `listSize` argument, but that feels bad for overloading a directive with data that isn't applicable for many fields.

Reply:
GraphQL doesn't really have directive overloads, does it? It's always going to clash on a duplicated directive definition. I'll use `@fakeList`

Reply:
I apparently need caffeine or something since my comments are all over the place. :) 

I mean more that you could add another argument (in this case, overloading the available arguments) like: `@faker(type: Type!, listSize: ListSize)` where `ListSize` is just the input args you defined in the existing directive.

Reply:
To be clear, I _really_ don't like the idea, but it is one and one I wanted to at least touch on.

Reply:
I sticked with `@fakeList`. If we were to put lists in `@fake()`, we would probably need `@examples()` to be an argument as well for consistency but then it's not exclusive anymore with `@fake()`. Let's see how that works.